### ISSUE TITLE
Auto-correct the tips dialog

### ISSUE THREAD

**Issue by GoogleCodeExporter** [iamakshayv/foursquared#38]
```
"""
Automagic punctuation doesn't work when adding a tip.  No capitalized 
letters by default and when I typed you'll there wasn't an apostrophe added.  
Not sure if this is known and don't have time to check but didn't want to 
forget.
"""
```

Original issue reported on code.google.com by `jlapenna` on 13 Aug 2009 at 2:30

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 18c30b3b07.
```

Original comment by `jlapenna` on 17 Aug 2009 at 9:36
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 18c30b3b07.
```

Original comment by `jlapenna` on 5 Jan 2010 at 5:47
